src/Plugin/BlockSettings/EntityTypeBundleSelectionTrait.php

```diff
@@ -35,7 +35,7 @@ public function buildSelectionSettingsForm(&$form, FormStateInterface $form_stat
 
     $subform['entity_reference'] = [
       '#type' => 'fieldset',
-      '#title' => t('Entity reference settings'),
+      '#title' => $this->t('Entity reference settings'),
```

Comment:
I am not sure about changing function t to $this->t. please verify It works

Reply:
seem to be working locally. I'll test further.

Reply:
you can just remove the t(). We don't need it. This module has to be cleaned up anyways at a later point in time